src/pages/index.tsx

```diff
@@ -30,35 +31,21 @@ const IndexPage: FunctionComponent = ({sections}: any) => {
 
 export default IndexPage
 
-const digitalGardeningQuery = groq`
-*[_type == 'resource' && slug.current == "digital-gardening-for-developers"][0]{
-  title,
-  description,
-  'illustration': images[label == 'eggo'][0]{
-    url,
-    alt
-  },
-  'quote': content[title == 'quote'][0]{
-    description
-  },
-  resources[]{
-    title,
-    byline,
-    'name': content[title == 'name'][0].description,
-    'path': resources[]->[0].path,
-    'image': resources[]->[0].image
-  }
+const featureQuery = groq`
+{
+  'featureCallOut': ${digitalGardeningQuery}
```

Comment:
we can construct stacks of queries that resolve into named properties and are also used elsewhere like the features page!